--- src/command.c.orig	2014-04-27 08:45:59.191631943 -0400
+++ src/command.c	2014-04-27 08:59:37.594561788 -0400
@@ -72,6 +72,7 @@
 #include <time.h>
 #include <stdlib.h>
 #include <errno.h>
+#include <unistd.h>
 
 #define COMMAND_STATUS_STATE            "state"
 #define COMMAND_STATUS_REPEAT           "repeat"
@@ -326,6 +327,39 @@
 }
 
 static enum command_return
+handle_pulseoff(struct client *client,
+              G_GNUC_UNUSED int argc, G_GNUC_UNUSED char *argv[])
+{
+	client_printf(client,"Stop Pulse\n");
+	system("pacmd unload-module module-rtp-send");
+	return COMMAND_RETURN_OK;
+}
+
+
+static enum command_return
+handle_pulseon(struct client *client,
+              G_GNUC_UNUSED int argc, G_GNUC_UNUSED char *argv[])
+{
+	client_printf(client,"Start Pulse\n");
+	system("pacmd load-module module-rtp-send source=rtp.monitor mtu=16000 loop=1 rate=44100 port=5555 ttl=2 channels=2");
+	return COMMAND_RETURN_OK;
+}
+
+
+static enum command_return 
+handle_pulsesync(struct client *client,
+              G_GNUC_UNUSED int argc, G_GNUC_UNUSED char *argv[])
+{
+	client_printf(client,"Sync Pulse Audio RTP\n");
+	client_printf(client,"Sound will stop/start\n");
+	system("pacmd unload-module module-rtp-send");
+	system("pacmd load-module module-rtp-send source=rtp.monitor mtu=16000 loop=1 rate=44100 port=5555 ttl=2 channels=2");
+
+	return COMMAND_RETURN_OK;
+}
+
+
+static enum command_return
 handle_status(struct client *client,
 	      G_GNUC_UNUSED int argc, G_GNUC_UNUSED char *argv[])
 {
@@ -2046,6 +2080,9 @@
 	{ "previous", PERMISSION_CONTROL, 0, 0, handle_previous },
 	{ "prio", PERMISSION_CONTROL, 2, -1, handle_prio },
 	{ "prioid", PERMISSION_CONTROL, 2, -1, handle_prioid },
+	{ "pulseoff", PERMISSION_CONTROL, 0, 0, handle_pulseoff },
+	{ "pulseon", PERMISSION_CONTROL, 0, 0, handle_pulseon },
+	{ "pulsesync", PERMISSION_CONTROL, 0, 0, handle_pulsesync },
 	{ "random", PERMISSION_CONTROL, 1, 1, handle_random },
 	{ "readmessages", PERMISSION_READ, 0, 0, handle_read_messages },
 	{ "rename", PERMISSION_CONTROL, 2, 2, handle_rename },
